<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusyBeeGUI</class>
 <widget class="QMainWindow" name="BusyBeeGUI">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BusyBee</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">
position: relative;
width: 1440px;
height: 1024px;

background: #FFFFFF;</string>
   </property>
   <widget class="QFrame" name="HeaderFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1231</width>
      <height>351</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(60, 73, 86);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="GenerateModel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>600</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1ABC9C;

/* Generate Model */

position: absolute;
width: 509px;
height: 76px;
left: 138px;
top: 24px;

font: 400 20pt &quot;Bitstream Vera Sans Serif&quot;;

color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>  Generate Model</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QFrame" name="selected">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>600</width>
       <height>11</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">position: absolute;
left: 0px;
top: 118px;

background: #C4C4C4;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="VisualizeModel">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>270</y>
       <width>600</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">position: absolute;
left: 0%;
right: 0%;
top: 0%;
bottom: 0%;

background: #3E5265;

/* Generate Model */

position: absolute;
width: 509px;
height: 76px;
left: 138px;
top: 24px;
font: 400 20pt &quot;Bitstream Vera Sans Serif&quot;;

color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>  Visualize Model</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Bee_icon">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/images/Bee-icon.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="pie_chart_icon">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>275</y>
       <width>56</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/images/pie_chart.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="BusyBee_icon">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>16</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#BusyBee_icon{
background-color: transparent;
border-image: url(:/images/images/bb_antenna.png);
background: none;
border: none;
background-repeat: none;
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_label">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>45</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 200 bold 45pt &quot;Helvetica&quot;;
color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Busy</string>
     </property>
    </widget>
    <widget class="QLabel" name="welcome_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>921</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 500 20pt &quot;Open Sans&quot;;
color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Welcome to BusyBee, the software that will allow you to easily&lt;/p&gt;&lt;p&gt;create and analyze hive data so you can keep your bees safe!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="total_nb_models_label">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>70</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bitstream Vera Sans Serif</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 400 italic 20pt &quot;Bitstream Vera Sans Serif&quot;;
color:rgb(211, 215, 207)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Total Number of &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Models:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="models_count">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>160</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 400 italic bold 26pt &quot;Bitstream Vera Sans Serif&quot;;
color:rgb(211, 215, 207)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 200 45pt &quot;Helvetica&quot;;
color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Bee</string>
     </property>
    </widget>
    <zorder>GenerateModel</zorder>
    <zorder>VisualizeModel</zorder>
    <zorder>selected</zorder>
    <zorder>Bee_icon</zorder>
    <zorder>pie_chart_icon</zorder>
    <zorder>BusyBee_icon</zorder>
    <zorder>title_label</zorder>
    <zorder>welcome_label</zorder>
    <zorder>total_nb_models_label</zorder>
    <zorder>models_count</zorder>
    <zorder>label</zorder>
   </widget>
   <widget class="QFrame" name="GenerateFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>1291</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="GenerateButton">
     <property name="geometry">
      <rect>
       <x>475</x>
       <y>360</y>
       <width>250</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">position: absolute;
width: 440px;
height: 95px;
left: 505px;
top: 903px;

background: #EC4C3C;

/* Generate */


font: 75 italic 30pt &quot;Bitstream Vera Serif&quot;;

color: #FFFFFF;
</string>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AnalyzeButton">
     <property name="geometry">
      <rect>
       <x>475</x>
       <y>360</y>
       <width>250</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">position: absolute;
width: 440px;
height: 95px;
left: 505px;
top: 903px;

background: #EC4C3C;

/* Generate */


font: 75 italic 30pt &quot;Bitstream Vera Serif&quot;;

color: #FFFFFF;
</string>
     </property>
     <property name="text">
      <string>Analyze</string>
     </property>
    </widget>
    <widget class="QPushButton" name="About_button">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>360</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#About_button {
background-color: transparent;
border-image: url(:images/images/BeeRex.png);
background: none;
border: none;
background-repeat: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="External_conditions_frame">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>351</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 2px;
selection-background-color: #32414B;

QFrame::selected {
    background-color: #32414B;
    border: 1px solid #1464A0;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="temp_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 400 14pt &quot;Bitstream Vera Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>Temperature: </string>
      </property>
     </widget>
     <widget class="QComboBox" name="season_combo">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Spring</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/spring_icon.png</normaloff>images/spring_icon.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Summer</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/sun_icon.png</normaloff>images/sun_icon.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Autumn</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/autumn_icon.png</normaloff>images/autumn_icon.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Winter</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/winter_icon.png</normaloff>images/winter_icon.png</iconset>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="season_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 400 14pt &quot;Bitstream Vera Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>Season: </string>
      </property>
     </widget>
     <widget class="QSlider" name="temperature_slider">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 10px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #66e, stop: 1 #bbf);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #bbf, stop: 1 #55f);
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="temperature_label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Bitstream Vera Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;60° Fahrenheit&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="temp_label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 14pt &quot;Bitstream Vera Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>Weather: </string>
      </property>
     </widget>
     <widget class="QComboBox" name="weather_combo">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Raining</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/rain_icon.ico</normaloff>images/rain_icon.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sunny</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/sun_icon.png</normaloff>images/sun_icon.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Windy</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/wind_icon.png</normaloff>images/wind_icon.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Snowy</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/snow_.ico</normaloff>images/snow_.ico</iconset>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QLabel" name="external_conditions_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 400 20pt &quot;Bitstream Vera Sans Serif&quot;;
color: rgb(32, 74, 135)</string>
     </property>
     <property name="text">
      <string>External Conditions:</string>
     </property>
    </widget>
    <widget class="QLabel" name="hive_information_label">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 400 20pt &quot;Bitstream Vera Sans Serif&quot;;
color: rgb(32, 74, 135)</string>
     </property>
     <property name="text">
      <string>Hive Information:</string>
     </property>
    </widget>
    <widget class="QFrame" name="hive_information_frame">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>70</y>
       <width>361</width>
       <height>261</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="nb_hives_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 14pt &quot;Bitstream Vera Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>Number of Hives:</string>
      </property>
     </widget>
     <widget class="QLabel" name="avg_size_hive_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 13pt &quot;Bitstream Vera Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>Average Colony Size/Hive:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="nb_hives_spinbox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="colony_size_slider">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 10px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #66e, stop: 1 #bbf);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #bbf, stop: 1 #55f);
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="colony_size_label">
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>170</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Bitstream Vera Serif&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Bees&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="colony_size_spinbox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="location_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="location_label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 14pt &quot;Bitstream Vera Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
     <zorder>location_label_2</zorder>
     <zorder>location_label</zorder>
     <zorder>nb_hives_label</zorder>
     <zorder>avg_size_hive_label</zorder>
     <zorder>nb_hives_spinbox</zorder>
     <zorder>colony_size_slider</zorder>
     <zorder>colony_size_label</zorder>
     <zorder>colony_size_spinbox</zorder>
    </widget>
    <widget class="QPushButton" name="default_button">
     <property name="geometry">
      <rect>
       <x>567</x>
       <y>170</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#default_button{
border-image: url(:/images/images/default_icon.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="random_button">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>260</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#random_button{
border-image: url(:/images/images/random_icon.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="const_pop_check_box">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>120</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="swarm_label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 400 bold 10pt &quot;Bitstream Vera Sans Serif&quot;;
color: rgb(32, 74, 135)</string>
     </property>
     <property name="text">
      <string>Swarm Mode:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="swarm_check_box">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>60</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="const_pop_label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>120</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 400 bold 10pt &quot;Bitstream Vera Sans Serif&quot;;
color: rgb(32, 74, 135)</string>
     </property>
     <property name="text">
      <string>Constant Population:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="models_list">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>691</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget::item {
    font: 30pt &quot;Bitstream Vera Sans Serif&quot;;
}

QListWidget::item:alternate {
    background: #EEEEEE;
}

QListWidget::item:selected {
    border: 1px solid #6a6ea9;
}

QListWidget::item:selected:!active {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #ABAFE5, stop: 1 #8588B2);
}

QListView::item:selected:active {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #6a6ea9, stop: 1 #888dd9);
}

QListWidget::item:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #FAFBFE, stop: 1 #DCDEF1);
}</string>
     </property>
    </widget>
    <zorder>random_button</zorder>
    <zorder>default_button</zorder>
    <zorder>GenerateButton</zorder>
    <zorder>AnalyzeButton</zorder>
    <zorder>About_button</zorder>
    <zorder>External_conditions_frame</zorder>
    <zorder>external_conditions_label</zorder>
    <zorder>hive_information_label</zorder>
    <zorder>hive_information_frame</zorder>
    <zorder>const_pop_check_box</zorder>
    <zorder>swarm_label</zorder>
    <zorder>swarm_check_box</zorder>
    <zorder>const_pop_label</zorder>
    <zorder>models_list</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
